### PR TITLE
New Connect page & footer change

### PR DESCRIPTION
**Pull request by krisfreedain**
Signed-off-by: Kris Freedain <freedak@amazon.com>

### Description
Created a new Connect page and modified the footer - allows the community to connect with the project in a number of ways.
 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by krisfreedain** [comment]
@stockholmux & @elfisher - please take a look. thanks

File: connect.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Connect with the Community
+---
+
+## Connect with the Community
+
+---
+
+There are several ways to connect with the OpenSearch community.
+
+* OpenSearch has [active forums](https://discuss.opendistrocommunity.dev/), which are the fastest way to get help using, troubleshooting, and developing with OpenSearch. As a bonus, questions and discussions on the forums are easy for others to learn from in the future.
```
Comment (elfisher) [inline connect.markdown:13]: Let's format these as paragraphs without bullets. We may even want to add headings.

File: connect.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Connect with the Community
+
+---
+
+There are several ways to connect with the OpenSearch community.
+
+* OpenSearch has [active forums](https://discuss.opendistrocommunity.dev/), which are the fastest way to get help using, troubleshooting, and developing with OpenSearch. As a bonus, questions and discussions on the forums are easy for others to learn from in the future. 
+* If you want to share a feature idea or submit a bug report, open an issue in the relevant repo in [the GitHub project](https://github.com/opensearch-project). 
+* If you want to present or have a question for an upcoming [community meeting](https://opensearch.org/events/), add a comment to the agenda for that meeting.
```
Comment (elfisher) [inline connect.markdown:15]: Let's move this up next to the forum point.

File: connect.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Businesses partnering in the development and advancement of OpenSearch can propose to [add themselves](https://github.com/opensearch-project/project-website#adding-to-the-partners-page) to the [partners](https://opensearch.org/partners/) page with a pull request.
+* To feature a plug-in or project you developed with OpenSearch, add it to the [community projects](https://opensearch.org/community_projects) page with a pull request.
```
Comment (elfisher) [inline connect.markdown:17]: let's move this below the feature request point.

File: connect.markdown [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* Information about using the OpenSearch brand and logo are available on the [Brand Guidelines](https://opensearch.org/brand.html) page.
```
Comment (elfisher) [inline connect.markdown:18]: let's move this below the partner point

File: connect.markdown [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+* To report or inquire about a potential security vulnerability, see the [Reporting a Vulnerability guidelines](https://github.com/opensearch-project/.github/blob/main/SECURITY.md).
```
Comment (elfisher) [inline connect.markdown:19]: @stockholmux aside from bolding, what could be done to make the security point more prominent. Moving it to the top?

**Review by elfisher — CHANGES_REQUESTED** [review]
Overall the content looks good. There is some formatting that would help make the page more readable. Let's first start with removing the bullet points and use paragraph formats. I made some recommendations on order so that similar content is grouped together.

File: _data/footer.yml [hunk 6]
```diff
@@ -36,7 +36,7 @@ columns:
         title: OpenSearch Disambiguation
         url: '/disambiguation.html'
   -
-    title: 'Connect'
+    title: 'Contact'
```
[the remark this answers is not in the document]
Comment (elfisher) [reply _data/footer.yml:39]: I now see why this was changed. My recommendation is "Contact Us"

**Comment by krisfreedain** [comment]
@elfisher - I've worked through requested updates and pushed the new files up for review/approval. thanks

**Review by elfisher — APPROVED** [review]

**Comment by elfisher** [comment]
> @elfisher - I've worked through requested updates and pushed the new files up for review/approval. thanks

This LGTM! Approved.

**Review by stockholmux — APPROVED** [review]

**Pull request merged**